packages/catalog-process/test/getEserviceById.test.ts

```diff
@@ -33,6 +33,7 @@ describe("get eservice by id", () => {
       id: generateId(),
       name: "eservice 001",
       descriptors: [descriptor1],
+      isDelegable: true,
```

Comment:
Interesting, I didn not add the previous flag here, could you add also `isSignalHubEnabled`? Also in the `getEservices` below

Reply:
Do you mean in esevice2 and eservice3? Do we really need that flag? I added `isDelegable` to the first one only to avoid having undefined there. If I have to add `isSignalHubEnabled` in every eservice, maybe it's better just to set that flag in the mockEservice.

Reply:
No sorry I meant in the other test, where it's also missing, I'm gonna comment there directly :)